### ISSUE TITLE
Allow `pkg_system_requirements()` to accept many packages

### ISSUE THREAD

**Issue by schloerke** [r-lib/pak#281]
If I have a handful of packages to install on GHA, I can map over them and individually call `pkg_system_requirements()` to make sure that I don't miss a requirement. 

```r
# current / slow / but works!
lapply(pkgs, function(pkg) {
  pak::pkg_system_requirements(pkg, execute = TRUE)
})
```

This slows down when I have to execute many of the same system requirement commands over and over.

It would be great if I could provide a vector of packages and `pak` would execute them using unique exec statements.

```r
# Want...
pak::pkg_system_requirements(pkgs, execute = TRUE)
```

**Comment by riccardoporreca** [comment]
@schloerke, this feature was implemented in #328.

**Comment by schloerke** [comment]
@riccardoporreca Do you know when this will be distributed to the stable release?
